quickstart/dedicated/Create a Model Bundle.ipynb

```diff
@@ -0,0 +1,388 @@
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<style>:root { --jp-notebook-max-width: 100% !important; }</style>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from IPython.display import display, HTML\n",
+    "display(HTML(\"<style>:root { --jp-notebook-max-width: 100% !important; }</style>\"))\n",
+    "import json\n",
+    "import os\n",
+    "from dotenv import load_dotenv\n",
+    "import pprint\n",
+    "load_dotenv()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "a252b07f-c85f-44a9-8185-505a41fbf6cf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from snsdk import SnSdk"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "02d004f8-6ff1-44ad-b33d-601d71528243",
+   "metadata": {},
+   "source": [
+    "## 2. Set up environment connector"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fd0b3583-b31f-45c6-87b6-b40a58bee55e",
+   "metadata": {},
+   "source": [
+    "Connects to the remote dedicated environment using the variables defined in `.env`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "772356ae-30a7-4780-a29e-9a71796cb0fb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sn_env = SnSdk(host_url=os.getenv(\"SAMBASTUDIO_HOST_NAME\"), \n",
+    "                   access_key=os.getenv(\"SAMBASTUDIO_ACCESS_KEY\"), \n",
+    "                   tenant_id=os.getenv(\"SAMBASTUDIO_TENANT_NAME\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cb162fb2-8da7-4a1b-8c96-762188c49888",
+   "metadata": {},
+   "source": [
+    "## 3. Select models to bundle"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f8003334-9e57-4b0d-8e77-9ab25fceae6c",
+   "metadata": {},
+   "source": [
+    "#### List models"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ebdbca6-b4d9-4b13-8075-97ade54ecb2c",
+   "metadata": {},
+   "source": [
+    "Get the complete list of models. This includes models that are  \n",
+    "  - actually available\n",
+    "  - still in the process of uploading\n",
+    "  - exist in a remote storage from which they can be made available\n",
+    "  - not in a usable state"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "08d1e30f-540a-4843-8ab1-7d01cfb406b7",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "139"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "models = sn_env.list_models()[\"models\"]\n",
```

Comment:
You can list only available model. I would like to know the reason behind loading all models and then filter it out.